demo/launch/run.launch.py

```diff
@@ -17,6 +17,11 @@ def generate_launch_description():
         executable=LaunchConfiguration("exe"),
         output="screen",
         parameters=[
+            os.path.join(
+                get_package_share_directory("moveit_task_constructor_demo"),
+                "config",
+                "panda_config.yaml",
+                ),
```

Comment:
Are you sure this works?
As far as I know, parameters should be a list of dictionaries (representing the parameter values).